### PR TITLE
Add support for Windows on Arm64 (WoA) build

### PR DESCRIPTION
**Pull request by chirontt**
The environment triplet for WoA build is: win32/win32/aarch64

### CODE REVIEW

File: eclipse.platform.releng.tychoeclipsebuilder/eclipse/publishingFiles/testManifest.xml [hunk 1]
```diff
@@ -9,6 +9,10 @@
     </zipType>
 
     <zipType name="Eclipse SDK">
+      <platform
```
Comment (akurtakov) [inline eclipse.platform.releng.tychoeclipsebuilder/eclipse/publishingFiles/testManifest.xml:12]: This one can't be produced as we can't rebuild natives on source change.

File: eclipse.platform.releng.tychoeclipsebuilder/eclipse/publishingFiles/testManifest.xml [hunk 2]
```diff
@@ -58,6 +62,10 @@
     </zipType>
 
     <zipType name="Platform Runtime Binary">
+      <platform
```
Comment (akurtakov) [inline eclipse.platform.releng.tychoeclipsebuilder/eclipse/publishingFiles/testManifest.xml:65]: Same.

File: eclipse.platform.releng.tychoeclipsebuilder/eclipse/publishingFiles/testManifest.xml [hunk 3]
```diff
@@ -100,6 +108,10 @@
     </zipType>
 
     <zipType name="SWT">
+      <platform
```
Comment (akurtakov) [inline eclipse.platform.releng.tychoeclipsebuilder/eclipse/publishingFiles/testManifest.xml:111]: Same.

File: eclipse.platform.releng.tychoeclipsebuilder/equinox/buildConfigs/equinox-launchers/build.xml [hunk 4]
```diff
@@ -13,6 +13,7 @@
 
   <target name="equinoxlaunchers">
     <property name="archiveRoot" value="launchers" />
+    <buildRepos os="win32" ws="win32" arch="aarch64" archiveName="${archiveRoot}-win32.win32.aarch64.${buildId}.zip" />
```
Comment (akurtakov) [inline eclipse.platform.releng.tychoeclipsebuilder/equinox/buildConfigs/equinox-launchers/build.xml:16]: Same.

File: eclipse.platform.releng.tychoeclipsebuilder/equinox/publishingFiles/testManifest.xml [hunk 5]
```diff
@@ -236,6 +236,11 @@
     </zipType>
 
     <zipType name="Launchers">
+      <platform
```
Comment (akurtakov) [inline eclipse.platform.releng.tychoeclipsebuilder/equinox/publishingFiles/testManifest.xml:239]: Same.

**Review by akurtakov — CHANGES_REQUESTED** [review]
We can not include native win32 aarch fragments in features unless we have a builder in our JIPPs allowing to rebuild them on e.g. source change in SWT. Until this happens we can only have sources that can't end up in deliverables.

**Comment by chirontt** [comment]
PR withdrawn, as build infrastructure is not yet in place to support building Windows on Arm64 artifacts.

**Comment by akurtakov** [comment]
Part of the patch e.g. eclipse-platform-parent/pom.xml one was good as it is first step towards doing any work in other repos. FWIW, linux/aarch64 sources went in partially and Fedora project used them to provide actual deliverables for couple of years.

**Pull request closed without merging**